src/mgr/Mgr.cc

```diff
@@ -553,8 +553,8 @@ bool Mgr::ms_dispatch2(const ref_t<Message>& m)
       handle_mon_map();
       break;
     case CEPH_MSG_FS_MAP:
-      py_module_registry->notify_all("fs_map", "");
       handle_fs_map(ref_cast<MFSMap>(m));
+      py_module_registry->notify_all("fs_map", "");
```

Comment:
This should be in a separate commit with explanation.

Reply:
(Looks like you figured out the problem with point 1. in the mail you sent.)

I would also send this fix as a separate PR so it can get in sooner.

Reply:
@batrick I thought I had figured out the problem, but doesn't look like so. The `fs_map` object is still undefined when the `clog` notification type is handled by the plugin.
Or maybe I turned too many knobs at once while changing the handling of notifications from`fs_map` to `clog`. The plugin probably needs to handle the `fs_map` notification to save the `fs_map` and then act on it when the `clog` notification is delivered.

Reply:
@batrick This was the missing scope specifier (`self`) prefix to the `fs_map` member that was biting me.